diff --git a/node_modules/@radix-ui/react-radio-group/dist/index.d.mts b/node_modules/@radix-ui/react-radio-group/dist/index.d.mts
index 466f8bb..9b6e502 100644
--- a/node_modules/@radix-ui/react-radio-group/dist/index.d.mts
+++ b/node_modules/@radix-ui/react-radio-group/dist/index.d.mts
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { Primitive } from '@radix-ui/react-primitive';
 import * as RovingFocusGroup from '@radix-ui/react-roving-focus';
+import { Gender, AccessRoles } from '@/utils/interfaces';
 
 type Scope<C = any> = {
     [scopeName: string]: React.Context<C>[];
@@ -35,8 +36,8 @@ type RadioGroupContextValue = {
     name?: string;
     required: boolean;
     disabled: boolean;
-    value?: string;
-    onValueChange(value: string): void;
+    value?: string | Gender | AccessRoles | boolean;
+    onValueChange(value: string | Gender | AccessRoles | boolean): void;
 };
 type RovingFocusGroupProps = React.ComponentPropsWithoutRef<typeof RovingFocusGroup.Root>;
 type PrimitiveDivProps = React.ComponentPropsWithoutRef<typeof Primitive.div>;
@@ -54,7 +55,7 @@ interface RadioGroupProps extends PrimitiveDivProps {
 declare const RadioGroup: React.ForwardRefExoticComponent<RadioGroupProps & React.RefAttributes<HTMLDivElement>>;
 type RadioProps = React.ComponentPropsWithoutRef<typeof Radio>;
 interface RadioGroupItemProps extends Omit<RadioProps, 'onCheck' | 'name'> {
-    value: string;
+    value: string | Gender | AccessRoles | boolean;
 }
 declare const RadioGroupItem: React.ForwardRefExoticComponent<RadioGroupItemProps & React.RefAttributes<HTMLButtonElement>>;
 type RadioIndicatorProps = React.ComponentPropsWithoutRef<typeof RadioIndicator>;
